packages/should-forward-prop/README.md

```diff
@@ -30,20 +21,20 @@ To exclude other custom props not included in Styled System, use the `createShou
 ```js
 import styled from '@emotion/styled'
 import { space, color } from 'styled-system'
-import { createShouldForwardProp, props } from '@styled-system/should-forward-prop'
+import {
+  createShouldForwardProp,
+  props,
+} from '@styled-system/should-forward-prop'
 
-const shouldForwardProp = createShouldForwardProp([
-  ...props,
-  'd',
-  'x'
-])
+const shouldForwardProp = createShouldForwardProp([...props, 'd', 'x'])
 
 const Box = styled('div', {
-  shouldForwardProp
-})(props => ({
-  display: props.d,
-  fontWeight: props.x ? 'bold' : null,
-}),
+  shouldForwardProp,
+})(
+  props => ({
```

Comment:
Sorry, let me think on this PR for a bit. I feel like prettier really degrades the quality of the code examples in places like this

Reply:
I see what you mean but it’s pretty minor and I’d argue that these days most people are more familiar with prettier formatted code, and the benefit of formatting across the readme’s in general would be worth it

Reply:
Perhaps I can pose it another way: Would it not make sense for the examples to be formatted in the same way as they would mostly likely end up appearing when they're used? 🙂 (I am making an assumption that most people using theme-ui format their code with Prettier).

Reply:
@jxnblk just pinging you on this one as it's been hanging around for a bit